### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by Alex237**
Add french translation for ODIN

### CODE REVIEW

**Comment by dgearhart** [comment]
@Alex237 we just recently updated our "posix_locale" usgae
If you change value to "fr_FR.UTF-8", you should be good to go
Thanks again

File: locales/fr-FR.json [hunk 1]
```diff
@@ -0,0 +1,997 @@
+{
+  "posix_locale": "fr_FR.utf8",
```
Comment (dgearhart) [inline locales/fr-FR.json]: change to "fr_FR.UTF-8"
Apologies for the confusion - I just updated README today

Comment (Alex237) [inline locales/fr-FR.json]: Done, you can now check it

**Comment by kevinkreiser** [comment]
@Alex237 @dgearhart appologies i forgot to update the README when I changed that. sorry for the troubles! and thanks for the excellent work!

**Comment by Alex237** [comment]
You are welcolme. i have merged your repository into mine

**Pull request merged**